docs/section-metadata.md

```diff
@@ -231,21 +231,27 @@ Narrow style puts the content in a section that is fixed to **610px**.
 Adds a divider before or after the section.
 
 
-| Divider |                                |
-| ------- | ------------------------------ |
-| before  | add divider before the section |
-| after   | add divider after the section  |
+| Divider   |                                  |
+|-----------|----------------------------------|
+| primary   | add divider with grey color 50%  |
+| secondary | add divider with blue color 34px |
 
 **_Example_**
 
 
-| Section Metadata |       |
-| ---------------- | ----- |
-| divider          | after |
-
+| Section Metadata |                  |
+| ---------------- |------------------|
+| divider          | primary or _empty_ |
```

Comment:
`secondary or _empty_`?
As primary seems to be the default option when empty

Reply:
It is the example table. Secondary example comes next